.github/workflows/xpu.yml

```diff
@@ -22,24 +22,6 @@ jobs:
       curr_branch: ${{ github.head_ref || github.ref_name }}
       curr_ref_type: ${{ github.ref_type }}
 
-  linux-jammy-xpu-py3_9-build:
```

Comment:
For reviewer's information: Drop the Intel GPU development bundle support as the older bundle has been out-of-date.